scripts/assets/kdoc_validity_exemptions.textproto

```diff
@@ -573,6 +573,7 @@ exempted_file_path: "domain/src/test/java/org/oppia/android/domain/topic/StoryPr
 exempted_file_path: "domain/src/test/java/org/oppia/android/domain/topic/TopicControllerTest.kt"
 exempted_file_path: "domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt"
 exempted_file_path: "domain/src/test/java/org/oppia/android/domain/util/StateRetrieverTest.kt"
+exempted_file_path: "instrumentation/src/java/org/oppia/android/instrumentation/TestApplicationComponent.kt"
```

Comment:
ApplicationComponent didn't have any KDoc so, I've added this to exemption too

Reply:
We added the previous file in exempted because there is no doc, but it should have and soon we will create to issues to fill those. 
As for this PR we can add doc and remove file from here.

Reply:
I think we are exempting kdocs for TestApplicationComponent interface
@sparsh1212 can you confirm